tests/trx_generator/trx_generator.hpp

```diff
@@ -15,8 +18,66 @@ namespace eosio::testing {
       fc::crypto::private_key _signer;
    };
 
+   struct action_pair_w_keys {
+      action_pair_w_keys(eosio::chain::action first_action, eosio::chain::action second_action, fc::crypto::private_key first_act_signer, fc::crypto::private_key second_act_signer)
+            : _first_act(first_action), _second_act(second_action), _first_act_priv_key(first_act_signer), _second_act_priv_key(second_act_signer) {}
+
+      eosio::chain::action _first_act;
+      eosio::chain::action _second_act;
+      fc::crypto::private_key _first_act_priv_key;
+      fc::crypto::private_key _second_act_priv_key;
+   };
+
+   struct account_name_generator {
```

Comment:
Add some unittests for this in `trx_generator_tests.cpp`

Reply:
Addressed: dd4fb9143fe